nifi-nar-bundles/nifi-update-attribute-bundle/nifi-update-attribute-processor/src/test/java/org/apache/nifi/update/attributes/TestUpdateAttribute.java

```diff
@@ -43,6 +45,7 @@
 
     final static private String TEST_CONTENT = "THIS IS TEST CONTENT";
 
+
```

Comment:
nit-pick: all that was changed to this file was new empty lines were added. Could these be removed?